resources/scss/_dashboard.scss

```diff
@@ -18,44 +18,15 @@
         }
     }
 
-    .loadoutCardContainer {
```

Comment:
This is used on the homepage. The styles are all wonky on the homepage without it.

<img width="1034" alt="Screenshot_9_7_20__12_24_PM" src="https://user-images.githubusercontent.com/1449463/92409103-194f9d80-f105-11ea-98db-8146b13d748c.png">